ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/LedgerDB/InMemory.hs

```diff
@@ -275,7 +275,7 @@ csBlockConfig = csBlockConfig' . csSecParam
 csBlockConfig' :: SecurityParam -> LedgerDbCfg (LedgerState TestBlock)
 csBlockConfig' secParam = LedgerDbCfg {
       ledgerDbCfgSecParam = secParam
-    , ledgerDbCfg         = HardFork.defaultEraParams secParam slotLength
+    , ledgerDbCfg         = testBlockLedgerConfigFrom $ HardFork.defaultEraParams secParam slotLength
```

Comment:
Style nitpick: line length

Reply:
Is there a size limit used at IOG? Which is it?

Reply:
I'm not sure what the proper style would be, I think maybe:
```haskell
    , ledgerDbCfg         =   testBlockLedgerConfigFrom
                            $ HardFork.defaultEraParams secParam slotLength
```
or
```haskell
    , ledgerDbCfg         =
          testBlockLedgerConfigFrom
        $ HardFork.defaultEraParams secParam slotLength
```
(By the way, this anti-indentation of `$`, I really cannot. It's not a matter of style, I just find it genuinely much harder to read and regularly rewrite expressions just to read the code before reverting my change.)